packages/block-library/src/file/style.scss

```diff
@@ -14,8 +14,8 @@
 		background: $dark-gray-700;
 		border-radius: 2em;
 		color: $white;
-		font-size: $default-font-size;
-		padding: 0.5em 1em;
+		font-size: 0.8em;
+		padding: 0.625em 1.25em;
```

Comment:
Again, there's no reason for the padding to change here since it was already using `em` units.

Reply:
Good spot, fixed :+1: